app/models/content_item.rb

```diff
@@ -1,14 +1,13 @@
 class ContentItem < ApplicationRecord
-
   validates_each :routes, :redirects do |record, attr, value|
     # This wording replicates the original Mongoid error message - we don't know if any downstream
     # consumers rely on parsing error messages at the moment
-    record.errors.add attr, "Value of type #{value.class} cannot be written to a field of type Array" unless value.nil? or value.respond_to?(:each)
+    record.errors.add attr, "Value of type #{value.class} cannot be written to a field of type Array" unless value.nil? || value.respond_to?(:each)
```

Comment:
I think the changes in this commit can be squashed into the previous one.

Maybe rubocop needs running after each commit.  I may give that a go and see what happens, as it'll make all the commits easier to review.